server/courselist.rb

```diff
@@ -100,7 +100,7 @@ class CourseList < Sinatra::Base
   config_hash[:studentdashboard] = "#{config_base}/studentdashboard.yml"
 
   # location for yml file describing the build.
-  config_hash[:build_file] = "#{config_base}/build.yml"
+  config_hash[:build_file] = "./server/local/build.yml"
```

Comment:
The build.yml is provided as part of the war file, not the configuration files.  The original reference was incorrect.